DEVELOPERS.md

```diff
@@ -48,6 +48,16 @@ A place to put scripts to be run via [runscript](https://django-extensions.readt
 
 The production database and backups are located at  `/var/lib/dokku/data/storage/opencodelists` on dokku3 (see also [deployment notes](DEPLOY.md)).
 
+The backups are created with the dumpdata management command (`deploy/bin/backup.sh`).
```

Comment:
Might be worth noting that these are backups of the core (default) db only, and not the coding system databases (which incidentally, are not backed up at all (but can theoretically be re-created from the coding system data files))

Reply:
Noted! Thank you 🙂